prod-refractr.yml

```diff
@@ -366,6 +366,7 @@ refracts:
 - www.mozilla.org/:
   # bug 1598043
   - mozilla.ch
+  - www.mozilla.ch
```

Comment:
I looks like refractr is returning a 404 for this address currently, this seems like a very reasonable change to resolve that issue.  I did notice that www.mozilla.ch has a cname to mozilla.ch, which does feel a little odd, but seems to work technically.

Reply:
Yep the CNAME is mentioned explicitly in the ticket. https://bugzilla.mozilla.org/show_bug.cgi?id=1598043#c7